### ISSUE TITLE
Question: Getting `AxiosError: Request failed with status 500` 

### ISSUE THREAD

**Issue by certrik** [ninoseki/mihari#478]
I have defined shodan api key with `export SHODAN_API_KEY=<MY API KEY>` and using 
```
title: <something> # String (required)
description: Tracking new instances of <some cn-name> # String (required)

ignore_old_artifacts: true # Whether to ignore old artifacts from checking or not (Optional, defaults to true)
ignore_threshold: 0 # Number of days to define whether an artifact is old or not (Optional, defaults to 0)

queries: # Array<Hash> (required)
  - analyzer: shodan # String (required)
    query: ssl.cert.subject.cn:<some cn-name> # String (required)
```
as a rule and getting `AxiosError: Request failed with status 500` in the WebApp.
What could I possible done wrong. Probably a lot but need help.

**Comment by ninoseki** [comment]
Please share the error log. Otherwise I don't have a clue.

**Comment by certrik** [comment]
THX. I would ... but where do I find the log?

**Comment by ninoseki** [comment]
You can see the log in your terminal.

```bash
$ mihari web
Puma starting in single mode...
* Puma version: 5.6.4 (ruby 3.1.1-p18) ("Birdie's Version")
*  Min threads: 1
*  Max threads: 1
*  Environment: production
*          PID: 80029
* Listening on http://127.0.0.1:9292
* Listening on http://[::1]:9292
Use Ctrl-C to stop
```

**Comment by ninoseki** [comment]
Hmm, could you share the rule if you don't mind?
It it is confidential, you can use my email address. (You can find it in my profile)

**Comment by ninoseki** [comment]
Also please share the following information.

```
### **System Information:**

- OS: [e.g. Windows10]
- Ruby version: [e.g. 3.0]
- Mihari version: [e.g. 2.0.0]
```

**Comment by certrik** [comment]
Rule is not very confidential but I will send it you in privat.
- OS: 5.13.0-41-generic #46~20.04.1-Ubuntu SMP Wed Apr 20 13:16:21 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
- Ruby version: 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]
- Mihari version: 4.6.0

**Comment by ninoseki** [comment]
Thanks, I just fixed the issue by #482.
But your rule matches with 2,000+ results. I think it will cause a timeout error in the web application.
Mihari's web app uses [Puma](https://github.com/puma/puma) and its default timeout value is 60s. The timeout value is not configurable right now.

So I recommend to use CLI to run the rule.

```bash
mihari search your_rule.yml
```

**Comment by ninoseki** [comment]
FYI the fix is included in v4.6.1.
https://github.com/ninoseki/mihari/releases/tag/v4.6.1
